conversion.go

```diff
@@ -1,7 +1,82 @@
 // Package conversion TODO(kkweon): Write description
 package conversion
 
+import (
+	"encoding/binary"
+	"errors"
+	"math"
+)
+
+type Endian int
+
+const BigEndian Endian = 0
+const LittleEndian Endian = 1
+
+type Option struct {
+	Endian Endian
+}
+
 // IntToFloat32 converts int to float32
 func IntToFloat32(x int) (float32, error) {
 	return float32(x), nil
 }
+
+// Float32ToBytes converts float32 to []byte with length 4
+func Float32ToBytes(x float32, o *Option) ([]byte, error) {
+	b := make([]byte, 4)
+	ux := math.Float32bits(x)
+	if o == nil || o.Endian == LittleEndian {
+		binary.LittleEndian.PutUint32(b[:], ux)
```

Comment:
궁금해서 여기서 전부 왜 `b` 가 아니라 `b[:]` 인가요? 

```suggestion
		binary.LittleEndian.PutUint32(b[:], ux)
```

Reply:
제가 순간 array 라고 착각했었나 봅니다.